src/components/addCard/addCardSection.tsx

```diff
@@ -0,0 +1,46 @@
+import { useContext } from "react";
+import {
+  BankContext,
+  DateContext,
+  NameContext,
+  NumberContext,
+} from "../../contexts/cardInfo";
+import { bank } from "../../core/bank";
+import { AddCardForm } from "./addCardForm";
+import { BankMenu } from "../bankModal/bankMenu";
+import { Card } from "../@common/card/card";
+import { ModalBackDrop } from "../@common/modal/modalBackDrop";
+import { ModalBox } from "../@common/modal/modalBox";
+import { ModalContent } from "../@common/modal/modalContent";
+import { ModalTrigger } from "../@common/modal/modalTrigger";
+
+export function AddCardSection() {
+  const { cardNumber } = useContext(NumberContext);
+  const { month, year } = useContext(DateContext);
+  const { userName } = useContext(NameContext);
+  const { selectedItem, selectItem } = useContext(BankContext);
+
+  return (
+    <>
+      <Card
+        cardNumber={cardNumber}
+        month={month}
+        year={year}
+        userName={userName}
+        cardColor={{
+          bgColor: bank[selectedItem]?.color,
+          fontColor: bank[selectedItem]?.font,
+        }}
+        bank={bank[selectedItem]?.logoName}
+      />
+      <AddCardForm />
+      <ModalBox defaultOpen>
+        <ModalTrigger />
+        <ModalBackDrop />
+        <ModalContent>
+          <BankMenu selectItem={selectItem} />
+        </ModalContent>
+      </ModalBox>
```

Comment:
많이 찾아보셔서 아시겠지만, 
Compound Component는 하나의 Context를 공유하기 때문에,
반드시 Context Provider 내에서 사용되어야 한다는 특징이 있어요. 

따라서 컴포넌트 명에도 이를 드러내는 경우가 많습니다.


```suggestion
      <Modal defaultOpen>
        <Modal.Trigger />
        <Modal.BackDrop />
        <Modal.Content>
          <BankMenu selectItem={selectItem} />
        </Moda.lContent>
      </Modal>
```

요런 느낌으로요/~~

Reply:
아핫..~! 사실 저렇게 컴포넌트명을 지어야한다는 생각이 들었는데, 너무 대놓고 compound 패턴에 집착하는 것 처럼 보일까봐 고민하던 중이었습니다..!! 알려주셔서 감사합니다!!! 수정했어요👍👍😃

Reply:
👍🏻